bundle.Dockerfile

```diff
@@ -15,7 +15,7 @@ LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 # Labels for testing.
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
-LABEL com.redhat.openshift.versions=v4.16-v4.17
+LABEL com.redhat.openshift.versions=v4.14
```

Comment:
Hi @ashleyvjoy Please update this as below:
LABEL com.redhat.openshift.versions="v4.14"  since this is not a range.